app/styles/app/modules/popup.scss

```diff
@@ -64,6 +64,20 @@
   position: absolute;
   top: 1em;
   right: 1em;
+  padding: 0;
+
+  .icon {
+    margin: 0;
+  }
+}
+
+.modal-controls {
+  text-align: right;
+  margin-top: 2em;
```

Comment:
I would normally prefer `rem` units for stuff like this, but I do notice that directly above and elsewhere in this file there's a lot of usage of `em` units, so I think in this case it makes sense to leave this as-is for consistency. And I can also definitely imagine cases where it is preferable to use `em` units.

But it has me thinking about the overall unit usage strategy for the codebase. I'm not sure if we have any existing guidelines for that kind of thing, but if not it might be worth working as a team to come up with some.

Reply:
Oh, it's a good point! Thanks! I don't think there's any consistency wrt units in our current styles 😅But it totally makes sense to relate such dimensional values to base font size instead of immediate value 👍I also think that we should try to improve things as we go, so I'll definitely change this one.

Reply:
Cool, that sounds good to me 😄 And yep, right now there is no consistency on it haha

Reply:
Fixed ✔️